build.gradle

```diff
@@ -0,0 +1,24 @@
+plugins {
+    id "java-gradle-plugin"
+    id "maven-publish"
+}
+
+group = "org.gradle.java"
+version = "0.1.0"
+
+publishing {
+    repositories {
+        maven {
+            url "/tmp/localPlugins"
```

Comment:
@melix, I don't think it works with the new `plugins {}` syntax, does it? Maybe I need to manually set up a dependency resolution rule?